signup.html

```diff
@@ -6,53 +6,79 @@
   <title>판다마켓</title>
   <link rel="stylesheet" href="css/style-signup.css">
   <link rel="icon" href="image/title_ic.png">
+  <link rel="stylesheet" as="style"href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
 </head>
 <body>
   <header class="signup-header">
     <a href="index.html">
       <img src="image/Property 1=Variant3@3x.png" alt="판다마켓 메인페이지로 가기" width="396px" height="132px">
     </a>
   </header>
-  <main class="signup-main">
-    <section class="signup-section">
-      <label class="signup-label-font" for="email">이메일</label>
-      <input class="signup-layout" id="email" type="email" placeholder="이메일을 입력해주세요">
-    </section>
-    <section class="signup-section">
-      <label class="signup-label-font" for="Nname">닉네임</label>
-      <input class="signup-layout" id="Nname" type="text" placeholder="닉네임을 입력해주세요">
-    </section>
-    <section class="signup-section">
-      <label class="signup-label-font" for="password">비밀번호</label>
-      <div class="password-hidden-layout">
-        <input class="signup-layout" id="password" type="password" placeholder="비밀번호를 입력해주세요">
-        <img class="password-hidden cursor-pointer" src="image/btn_visibility_on_24px.png" alt="비밀번호 표시 변경">
+  <form action="" accept-charset="UTF-8" name="panda-signup" method="get">
+    <main class="signup-main">
```

Comment:
`main`에는 페이지의 컨텐츠들이 들어가는 태그이고 `form`은 회원가입 페이지에서 중요한 컨텐츠라고 볼 수 있어서 둘의 위치가 바뀌어야합니다.

```suggestion
<main class="signup-main">
  <form action="" accept-charset="UTF-8" name="panda-signup" method="get">

```